src/analyzer.py

```diff
@@ -138,3 +140,51 @@ def noun_phrase(input_text):
     for chunk in doc.noun_chunks:
         n_phrase_lst.append(str(chunk))
     return n_phrase_lst
+
+
+def senti_pos(tokens_column):
+    """Create column for positive words"""
+    # Start off with an empty list
+    display_series = []
+    # For each item in the already existing tokens column in the data frame,
+    for token_element in tokens_column:
+        words = create_word_list(token_element)
+        # Add the words at the end of the list to the display series since
+        # they have the most positive sentiment value
+        display_series.append(", ".join(words[len(words) - 3: len(words)]))
```

Comment:
Just a small suggestion: is this just looking for the last three words in the list? Might be worth looking into backward list index like `[::-1]`. Check out this link as well: https://stackoverflow.com/questions/47025356/reverse-index-in-a-list